src/components/Description/index.js

```diff
@@ -16,165 +16,191 @@ import fonts from 'EBI-Icon-fonts/fonts.css';
 
 const f = foundationPartial(ebiStyles, styles, theme, fonts);
 
-export const ParagraphWithCites = ({
```

Comment:
It might be worth renaming ParagraphWithCites to ParagraphsWithCitations for search-ability in future.